src/components/slider/slider.js

```diff
@@ -140,7 +140,15 @@ function SliderDirective($$rAF, $window, $mdAria, $mdUtil, $mdConstant, $mdThemi
       ngModelRender();
       redrawTicks();
     }
-    setTimeout(updateAll, 0);
+
+    // Wait for the slider dimensions to exist
+    var widthInterval = setInterval(function () {
+      var width = trackContainer[0].offsetWidth;
+      if (width > 0) {
+        updateAll();
+        clearInterval(widthInterval);
+      }
+    }, 250);
```

Comment:
@ThomasBurleson it was arbitrary, there might be a better way to detect when the slider is ready actually.